junifer/datagrabber/tests/test_aomic.py

```diff
@@ -0,0 +1,380 @@
+"""Provide tests for aomic1000."""
```

Comment:
can you place this file in `datagrabber/aomic/tests`? And can you split it in 3 files, one for each dataset?